wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_dropdowns.scss

```diff
@@ -155,7 +155,6 @@
     }
 
     // Styles for dropdowns which are also buttons e.g page editor
-
```

Comment:
These are the reverse of the earlier rule that added a space after a single line comment. I think for the space to be there the comments would need to be `/* */`.